beacon-chain/rpc/eth/node/handlers.go

```diff
@@ -37,14 +38,29 @@ func (s *Server) GetSyncStatus(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	isElOnline := s.ExecutionChainInfoFetcher.ExecutionClientConnected()
+	if !isElOnline {
+		isSyncing, err := s.ExecutionChainInfoFetcher.IsExecutionClientSyncing(ctx)
+		if !isSyncing {
+			if err != nil {
+				log.WithField("error", err.Error()).Debug("failed to get execution client sync status")
+			} else {
+				log.Debug("execution client is not syncing and not connected")
+			}
+		}
+		if isSyncing && err == nil {
+			isElOnline = true
+		}
+	}
+
 	headSlot := s.HeadFetcher.HeadSlot()
 	response := &structs.SyncStatusResponse{
 		Data: &structs.SyncStatusResponseData{
 			HeadSlot:     strconv.FormatUint(uint64(headSlot), 10),
 			SyncDistance: strconv.FormatUint(uint64(s.GenesisTimeFetcher.CurrentSlot()-headSlot), 10),
 			IsSyncing:    s.SyncChecker.Syncing(),
```

Comment:
Why not use:
```suggestion
			IsSyncing:    isSyncing,
```

Reply:
that's for consensus client syncing, isSyncing is for execution client